deployment-scripts/update-manifest.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+
+if [[ -z "$1" ]] || [[ -z "$2" ]]  || [[ -z "$3" ]]; then 
+  echo "Usage: update-manifest.sh <manifest-git-repo> <component-name> <image-uri>"
+  exit -1;
+fi
+
+
+git_url=$1
+comp=$2
+uri=$3
+
+set -euo pipefail
+
+if [[ ! -z "${GIT_DEPLOYMENT_KEY+x}" ]]; then 
+  git-set-creds-github.sh "$GIT_DEPLOYMENT_KEY"
+fi
+
+cd $DIR
+repo_name=$(echo $git_url| sed -e 's#.*/## ; s/.git//g')
```

Comment:
The last regex could probably be tightened with `$`. I _think_ we could make use of basename too if you guys think that would be cleaner. i.e.
```shell
repo_name=$(basename $git_url | s/.git$//')
```

Reply:
Note I'm not massively fussed. It's just an idea so let me know what you think.

Reply:
Nice creative idea. I'll also escape the dot in '.git'